src/components/StaffList/StaffList.scss

```diff
@@ -45,11 +45,10 @@
   margin: 0;
   padding: 0;
   list-style-type: none;
-  max-height: 300px;
+  max-height: 318px;
   overflow-y: auto;
 }
 
 .react-autosuggest__suggestion--highlighted {
   background-color: $light-gray;
 }
```

Comment:
Ugh, we have so much custom scss here. I suspect a lot of it could be replaced with bootstrap - like the width, the absolute position, the border stuff. Doesn't have to be cleaned up in this PR. But just a note that custom scss is more difficult to maintain. Please consider avoiding it when possible.

Reply:
spoke in person about custom css and 3rd party components